src/fs9p/dune

```diff
@@ -1,4 +1,4 @@
 (library
  (name fs9p)
  (wrapped false)
- (libraries protocol-9p mirage-flow))
+ (libraries protocol-9p mirage-flow result))
```

Comment:
I don't think you need this `result` package if you use a modern OCaml. 4.08+ (as the lower bound currently is) should be sufficient as `Result` has been present since then.